QuasistaticBrownianThermalNoise/QuasistaticBrownianThermalNoise.cpp

```diff
@@ -500,8 +540,31 @@ CylTransFunc::CylTransFunc(double coatThick, double halfCylThick):
     //kIso_AlGaAs #effective isotropic approx to x=0.92 AlGaAs
     //kIso_Ta2O5 #tantalum
     //kIso_FusedSilica #fused silica
-    mWhichCoatingYijkl = kAlGaAs;
+    mWhichCoatingYijkl =   kIso_Ta2O5;
     mWhichSubstrateYijkl = kIso_FusedSilica;
+      
+      
+    lossPhi_Iso_FusedSilica = 1.e-6; //Cole+ (2013)
+    //    lossPhi_Iso_Ta2O5 = 4.e-4; //Yamamoto+ (2006), PRD 74, 022002
+    // (good value??)
+    lossPhi_Iso_Ta2O5 = 2.3e-4; // Value from Hong + et. al.
+
+    lossPhi_Iso_AlGaAs = 2.5e-5; //Cole+ (2013)
+    lossPhi_AlGaAs = lossPhi_Iso_AlGaAs; //use same loss angle from Cole+(2013)
+      
+    phi11 = 2.5e-5;
+    phi12 = 2.5e-5;
+    phi44 = 2.5e-5;
+    
+      
+    lossPhi_Iso_FusedSilica_bulk  = lossPhi_Iso_FusedSilica;
+    lossPhi_Iso_FusedSilica_shear = lossPhi_Iso_FusedSilica;
+    
+    lossPhi_Ta2O5_bulk  = 0.00115;
+    lossPhi_Ta2O5_shear = 2.3e-4;
+      
+    lossPhi_Iso_AlGaAs_bulk  = lossPhi_Iso_AlGaAs;
+    lossPhi_Iso_AlGaAs_shear = lossPhi_Iso_AlGaAs;
```

Comment:
Same comment as above: shouldn't these loss angles have a bulk and shear loss that are different? If there are no such values in the literature, let's make a comment here that we don't have good numbers yet.